test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeControllerSpec.scala

```diff
@@ -945,6 +949,197 @@ class RepresenteeControllerSpec
 
     }
 
+    "handling requests to display the confirm person page" must {
+      def performAction(): Future[Result] = controller.confirmPerson()(FakeRequest())
+
+      behave like redirectToStartBehaviour(performAction)
+
+      behave like nonCapacitorOrPersonalRepBehaviour(performAction)
+
+      def test(
+        sessionData: SessionData,
+        expectedBackLink: Call,
+        expectReturnToSummaryLink: Boolean,
+        expectedPrepopulatedValue: Option[RepresenteeReferenceId]
+      ): Unit = {
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(sessionData)
+        }
+
+        checkPageIsDisplayed(
+          performAction(),
+          messageFromMessageKey("representeeConfirmPerson.title"),
+          doc => {
+            doc.select("#back").attr("href") shouldBe expectedBackLink.url
+            doc
+              .select("#name-question")
+              .text shouldBe (messageFromMessageKey("representeeConfirmPerson.summaryline1"))
+            doc
+              .select("#name-answer")
+              .text shouldBe "First Last"
```

Comment:
where does `"First Last"` come from? Maybe the name and the id should be passed in so it's clearer

Reply:
Done.

Reply:
not sure that you have. If you pass in `name: IndividualName` to the function then you can reference it here. "First Last" is a bit magic here

Reply:
Sorry, done now.